src/blocks/mod.rs

```diff
@@ -404,6 +409,23 @@ impl Block {
     }
 }
 
+impl Display for Block {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let work_render = if self.work.is_none() {
+            "No Work".to_string()
+        } else {
+            self.work.as_ref().unwrap().to_string()
+        };
+        let signature_render = if self.signature.is_none() {
+            "No Signature".to_string()
+        } else {
+            self.signature.as_ref().unwrap().to_string()
+        };
+        write!(f, "Block(Account: {}, Previous: {:?}, Balance: {}, Link: {:?}, Work: {}, Signature: {})",
```

Comment:
Are you running rustfmt? The IntelliJ plugin has settings that can automatically do rustfmt on save. Make sure you set `Use rustfmt instead of built-in formatter` too.

![image](https://user-images.githubusercontent.com/31338/113505933-1cb8e380-9585-11eb-8c72-eb19806c5b1b.png)

Reply:
I did, it probably reverted for some reason

Reply:
Btw, can we enforce this at CI level?

Reply:
Yes, I will do this today :)